src/mod/irc.mod/irc.c

```diff
@@ -1045,13 +1061,59 @@ static void flush_modes()
   }
 }
 
+static void tell_account_tracking_status(int idx, int details)
+{
+  struct capability *current;
+  int extjoin = 0, notify = 0, tag = 0, whox = use_354;
+  /* List status of account tracking. For 100% accuracy, this requires
+   * WHOX ability (354 messages) and the extended-join and account-notify
+   * capabilities to be enabled.
+   */
+  /* Check if CAPs are enabled */
+  current = cap;
+  while (current != NULL) {
+    if (!strcasecmp("extended-join", current->name) && current->enabled) {
+      extjoin = 1;
+    } else if (!strcasecmp("account-notify", current->name) && current->enabled) {
+      notify = 1;
+    } else if (!strcasecmp("account-tag", current->name) && current->enabled) {
+      tag = 1;
+    }
+    current = current->next;
+  }
+
+  if (whox && notify && extjoin) {
+    dprintf(idx, "%s", "    Account tracking: Perfect\n");
+  } else {
+    if (!details) {
+      dprintf(idx, "    Account tracking: Best-effort (Missing capabilities:%s%s%s%s)\n",
+                    whox ? "" : " WHOX", notify ? "" : " account-notify", extjoin ? "" : " extended-join",
+                    details ? "" : ", see .status all for details");
+    } else {
+      dprintf(idx, "    Account tracking: Best-effort\n");
+      if (!whox) {
+        dprintf(idx, "%s", "      - WHOX missing           => Accounts will not be known after Eggdrop joins a channel\n");
+      }
+      if (!notify) {
+        dprintf(idx, "%s", "      - account-notify missing => Accounts will not update immediately when users log in or out\n");
+      }
+      if (!extjoin) {
+        dprintf(idx, "%s", "      - extended-join missing  => Accounts will not be known immediately when a user joins\n");
+      }
+      if (tag) {
+        dprintf(idx, "%s", "      - account-tag present    => Accounts will update whenever someone messages a channel or this bot\n");
```

Comment:
I like this idea... maybe add the inverse comment as well, what *does* work? 
```
      if (!whox) {
        dprintf(idx, "%s", "      - WHOX missing           => Accounts will not be known after Eggdrop joins a channel\n");
      } else {
        dprintf(idx, "%s", "      - WHOX present           => Accounts will update when Eggdrop joins a channel\n");
      }
```
to give the user a full idea of what it does/doesn't do?